boards/nordic/nrf54h20dk/Kconfig.defconfig

```diff
@@ -14,3 +14,11 @@ config BT_CTLR
 	default y if BT
 
 endif # BOARD_NRF54H20DK_NRF54H20_CPURAD
+
+if BOARD_NRF54H20DK_NRF54H20_CPUPPR
+
+# As PPR has limited memory most of tests does not fit with asserts enabled.
+config ASSERT
+	default n
+
+endif # BOARD_NRF54H20DK_NRF54H20_CPUPPR
```

Comment:
why not enable this at soc level?, also, make it conditional to ztest maybe? like

```
config ASSERT
    default n if ZTEST
```

Reply:
right, done for soc level.
As for assert kconfig. I used to have like you suggested but changed based on that comment: https://github.com/zephyrproject-rtos/zephyr/pull/74297#discussion_r1642541482